api/types/events/events.go

```diff
@@ -1,33 +1,26 @@
 package events // import "github.com/docker/docker/api/types/events"
 
+// Type is used for event-types.
+type Type = string
+
+// List of known event types.
 const (
-	// BuilderEventType is the event type that the builder generates
-	BuilderEventType = "builder"
-	// ContainerEventType is the event type that containers generate
-	ContainerEventType = "container"
-	// DaemonEventType is the event type that daemon generate
-	DaemonEventType = "daemon"
-	// ImageEventType is the event type that images generate
-	ImageEventType = "image"
-	// NetworkEventType is the event type that networks generate
-	NetworkEventType = "network"
-	// PluginEventType is the event type that plugins generate
-	PluginEventType = "plugin"
-	// VolumeEventType is the event type that volumes generate
-	VolumeEventType = "volume"
-	// ServiceEventType is the event type that services generate
-	ServiceEventType = "service"
-	// NodeEventType is the event type that nodes generate
-	NodeEventType = "node"
-	// SecretEventType is the event type that secrets generate
-	SecretEventType = "secret"
-	// ConfigEventType is the event type that configs generate
-	ConfigEventType = "config"
+	BuilderEventType   Type = "builder"   // BuilderEventType is the event type that the builder generates.
+	ContainerEventType Type = "container" // ContainerEventType is the event type that containers generate.
+	DaemonEventType    Type = "daemon"    // DaemonEventType is the event type that daemon generate.
+	ImageEventType     Type = "image"     // ImageEventType is the event type that images generate.
+	NetworkEventType   Type = "network"   // NetworkEventType is the event type that networks generate.
+	PluginEventType    Type = "plugin"    // PluginEventType is the event type that plugins generate.
+	VolumeEventType    Type = "volume"    // VolumeEventType is the event type that volumes generate.
+	ServiceEventType   Type = "service"   // ServiceEventType is the event type that services generate.
+	NodeEventType      Type = "node"      // NodeEventType is the event type that nodes generate.
+	SecretEventType    Type = "secret"    // SecretEventType is the event type that secrets generate.
+	ConfigEventType    Type = "config"    // ConfigEventType is the event type that configs generate.
```

Comment:
Hm... I should've sorted this list while I was  changing this.

Let me get that sorted (pun intended) before we merge this 😅